tests/test_prob.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for the PennyLane qml.probs() measurement function.
+"""
+import pytest
+
+import numpy as np
+import pennylane as qml
+
+
+# make the test deterministic
+np.random.seed(42)
+
+
+@pytest.fixture
+def init_state(scope="session"):
+    """Fixture that creates an initial state"""
+    def _init_state(n):
+        """An initial state over n wires"""
+        state = np.random.random([2 ** n]) + np.random.random([2 ** n]) * 1j
+        state /= np.linalg.norm(state)
+        return state
+
+    return _init_state
```

Comment:
Maybe it would be nice to have this in the ``conftest.py``.

Reply:
True, that's a good point. I'm a bit hesitant to put it in the `conftest.py` file right away, as the conftest file is a bit... messy, and needs cleaning up (there are two many fixtures which aren't used or aren't needed).